### ISSUE TITLE
Use system icons under linux

### ISSUE THREAD

**Issue by adrianofoschi** [tibirna/qgit#13]
Is possibile to use system icons under linux?
Tipically adwaita under gnome and breeze under KDE Plasma. 

Thanks you

**Comment by tibirna** [comment]
Not presently.
I intended for a long time to better integrate qgit with the environment (essentially look and feel and session support). Supporting DE icons requires quite a bit of coding (detecting DE kind and version, extracting icon paths, gathering info on icon names etc.). It's doable but will require some time.
Thanks for the suggestion. I'll look into it when possible.

**Comment by peterhoeg** [comment]
I just came here to open an issue for the very same thing so there's definitely a +1 from here.

**Comment by filiperinaldi** [comment]
QGit targets Windows and MacOS. The same complications pointed by @F1ash in qt-virt-manager#19 applies to QGit.
BTW, I'm working on a patch to replace QGit's icons though whether Tibirna will accept it is another story ;-)
Check the branch **prosecco** for this and other changes I'm working on: https://github.com/filiperinaldi/qgit/tree/prosecco

**Comment by peterhoeg** [comment]
If the icons are referenced by their standard name and no icon by that name is supplied with qgit, Qt will load from the global theme at least on Linux. That means that the icons provided with qgit should be:

a) For non-linux platforms
b) If a standard icon does not exist
c) If a standard icon needs to be overridden

**Comment by mwoehlke** [comment]
I haven't checked (because I *have* an icon theme), but the impression I get from designer is that if you specify both a resource *and* a theme icon, you'll get the theme icon if available, and the resource otherwise. This would imply all that is needed is to specify the theme icons.

I have a branch for that, locally, but I'm waiting on my other PR (which is a build fix for my setup) to be accepted.

**Comment by mwoehlke** [comment]
https://github.com/mwoehlke/kgit/tree/use-theme-icons

Whee! Um... poorly... but mainly because git is going to fall flat on its face trying to deal with the "conflicts". I don't think the two branches *actually* trip over each other, aside from needing to touch code that is close enough together to give git fits...
